cmd/config.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cmd
+
+// GoogleSafeBrowsingConfig is the JSON config struct for the VA's use of the
+// Google Safe Browsing API.
+type GoogleSafeBrowsingConfig struct {
+	APIKey  string
+	DataDir string
+}
```

Comment:
I think the idea here is to start splitting out the config stuff from shell.go into its own file, which I think is a great idea.